src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/TempRepositoryTest.kt

```diff
@@ -119,17 +119,17 @@ internal class TempRepositoryTest : IntegrationTestBase() {
     assertThat(actual.personalSkillsAndInterests!!.updatedBy).isNotNull()
     assertThat(actual.personalSkillsAndInterests!!.updatedByDisplayName).isNotNull()
 
-    assertThat(actual.futureWorkInterestsEntity).isNotNull()
-    assertThat(actual.futureWorkInterestsEntity!!.id).isNotNull()
-    assertThat(actual.futureWorkInterestsEntity!!.reference).isNotNull()
-    assertThat(actual.futureWorkInterestsEntity!!.interests).hasSize(1)
-    assertThat(actual.futureWorkInterestsEntity!!.createdAt).isNotNull()
-    assertThat(actual.futureWorkInterestsEntity!!.createdAtPrison).isEqualTo(prisonId)
-    assertThat(actual.futureWorkInterestsEntity!!.createdBy).isNotNull()
-    assertThat(actual.futureWorkInterestsEntity!!.createdByDisplayName).isNotNull()
-    assertThat(actual.futureWorkInterestsEntity!!.updatedAt).isNotNull()
-    assertThat(actual.futureWorkInterestsEntity!!.updatedAtPrison).isEqualTo(prisonId)
-    assertThat(actual.futureWorkInterestsEntity!!.updatedBy).isNotNull()
-    assertThat(actual.futureWorkInterestsEntity!!.updatedByDisplayName).isNotNull()
+    assertThat(actual.futureWorkInterests).isNotNull()
+    assertThat(actual.futureWorkInterests!!.id).isNotNull()
+    assertThat(actual.futureWorkInterests!!.reference).isNotNull()
+    assertThat(actual.futureWorkInterests!!.interests).hasSize(1)
+    assertThat(actual.futureWorkInterests!!.createdAt).isNotNull()
+    assertThat(actual.futureWorkInterests!!.createdAtPrison).isEqualTo(prisonId)
+    assertThat(actual.futureWorkInterests!!.createdBy).isNotNull()
+    assertThat(actual.futureWorkInterests!!.createdByDisplayName).isNotNull()
+    assertThat(actual.futureWorkInterests!!.updatedAt).isNotNull()
+    assertThat(actual.futureWorkInterests!!.updatedAtPrison).isEqualTo(prisonId)
+    assertThat(actual.futureWorkInterests!!.updatedBy).isNotNull()
+    assertThat(actual.futureWorkInterests!!.updatedByDisplayName).isNotNull()
```

Comment:
I considered switching to the new entity custom assertions classes, but this test isn't hanging around for long.

Reply:
Thats fair 👍